bundled_scripts/ilo.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+import sys
+from copy import deepcopy
+
+import hpilo
+
+
+USER = os.environ['MANAGEMENT_USER_NAME']
```

Comment:
That's intentional :) You most certainly do not want to pass any sensible data (e.g. credentials) via script params. But I see your point, and I think that moving IP address argument to env var would be a good move here (these scripts are not meant to be launched by user directly, anyway).